RIOT-OS/RIOT
# pkg: ccn_lite update

* updates CCN-lite to newest master
* enables RONR (reactive opportunistic name-based routing) in example application
* automatically use the first interface in the example application

**Update**
latest CCN-lite version provides:
* make the cache replacement strategy configurable
* utility function to print the content store
* allow loopback packets
* updated prototype for interest sending function
* fixes off-by one error for Interest retransmissions
* set the default cache size on RIOT to 5 (defined via `CCNL_CACHE_SIZE`)
* check the maximum number of supported interfaces (currently 1)
* limit the number of PIT entries (20 per default)
* some fixes for Interest timeout handling
* use xtimer instead of CCN-lite timer
* fix race condition in PIT timeout handling
* implement GNRC netapi send function
